plugins/foreman_remote_execution/nightly/index.md

```diff
@@ -228,24 +228,17 @@ stands for variables which you define in second tab called Job.
 ![Job Template form](/plugins/foreman_remote_execution/{{page.version}}/job_template_form.png)
 
 Before we get to inputs, lets look at other fields in Job tab. First one is an
-autocomplete field called **Job name**. This is generic name of the Job, which
-might be implemented by more than one job template.  For example, one template
-may use `yum` to install a package, while the other uses `dnf`.  At invocation
-time you can choose which template to use.
-
-This is more useful when more than one provider is available. For example, you
-might have both an SSH and an Ansible template that implements the "Install
-Package" job. It turned out that having two names is so complicated
-that we decided to merge template and job name in some upcoming version.
-
-A description format field is a template for job name. Imagine you run job
-template named "Install package" 10 times, each time with different package
+autocomplete field called **Job category**. This groups job template together
+and makes orientation in huge number of template easier.
+
+A description format field is a template for job description. Imagine you run
+a template named "Install package" 10 times, each time with different package
 name specified by user input (more about this below). On job overview page
 you'd see the same "Install package" job and you'd have to inspect provided
 inputs to find the one that installed let's say httpd package. Using
-description format you set your own rules for naming which can use input
-values as part of job names. In our example install package example a good
-description format would be following
+description format you set your own rules for creating the description.
+Input values can be used as part of it. In our example install package example,
```

Comment:
This sentence has too many examples, maybe "A good description format for our package install example would be..."?